frontend/packages/topology/src/components/page/LimitExceededState.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import {
+  Button,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateSecondaryActions,
+  Title,
+} from '@patternfly/react-core';
+import { TopologyIcon } from '@patternfly/react-icons';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+type LimitExceededStateProps = {
+  onShowTopologyAnyway: () => void;
+};
+
+export const LimitExceededState: React.FC<LimitExceededStateProps> = ({ onShowTopologyAnyway }) => {
+  const { t } = useTranslation();
+  return (
+    <EmptyState>
+      <EmptyStateIcon variant="container" component={TopologyIcon} />
+      <Title headingLevel="h4" size="lg">
+        {t(`topology~Loading is taking longer than expected`)}
+      </Title>
+      <EmptyStateBody>
+        {t(
+          `topology~We noticed that it is taking a long time to visualize your application Topology. You can use Search to find specific resources or click Continue to keep waiting.`,
+        )}
+      </EmptyStateBody>
+      <Button variant="primary" component={(props) => <Link {...props} to="/search-page" />}>
```

Comment:
`Go to search` button redirect to the 404 page. Look like in 4.8 `/search-page` URL does not exist. We have `/search` in 4.8 for the search page.
![Peek 2022-09-19 20-33](https://user-images.githubusercontent.com/2561818/191050486-2225c01b-273d-4f59-a1d0-50eda63716f1.gif)

Reply:
fixed